create_bot_knowledge.py

```diff
@@ -32,14 +32,14 @@
 _LOGGER = logging.getLogger(__name__)
 logging.basicConfig(level=logging.INFO)
 
-_GITHUB_ACCESS_TOKEN = os.getenv("GITHUB_ACCESS_TOKEN")
+_GITHUB_ACCESS_TOKEN = 'a065981b6034f15156bbba651bea2687c4934194'
```

Comment:
This is a major security incident

Reply:
yeah i should be arrested for this horrendous mistake by cyber police... fortunatelly francesco was quick enough to point it out, it should be now removed and the token is no more valid